static/i18n/en/app.json

```diff
@@ -196,7 +196,7 @@
     "successDescription_plural": "Your accounts have been created.",
     "createNewAccount": {
       "title": "Add a new account",
-      "noOperationOnLastAccount": "No transactions found on your last new account <1><0>{{accountName}}</0></1>. You can add a new account after you've started transacting on that account.",
+      "noOperationOnLastAccount": "There are no transactions on your last created <1><0>{{accountName}}</0></1> account. You must fund it before creating another account.",
```

Comment:
There are no transactions on your last created <1><0>{{accountName}}</0></1> account. You must first receive a transaction on that account before you can add a new one.

Reply:
i feel the expression `receive a transaction` is wrong, no? shouldn't it be more `cryptoassets` ?

you receive funds / cryptoassets via a transaction 🤔


especially the first sentence already say "transaction" so it would be clear to also say you need to receive funds

(or whatever alternative you have to say "funds")